packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4462,13 +4462,17 @@ abstract class RenderObjectElement extends Element {
   @override
   void deactivate() {
     super.deactivate();
-    assert(!renderObject.attached);
+    assert(!renderObject.attached,
+      'A RenderObject was still attached when attempting to deactivate its '
+      'RenderObjectElement: ${renderObject.toString()}');
```

Comment:
the toString is implied

Reply:
Duh.  I had toStringDeep there before, and wasn't thinking when I just removed the "Deep".  Fixed.